### ISSUE TITLE
[Intro to iOS] Project 2 - Review my app

### ISSUE THREAD

**Issue by pthor** [pthor/ios_yelp_swift#1]
my yelp app is complete, please review. /cc @codepathreview
I may update tonight if I get more time to work on the project

**Comment by codepathreview** [comment]
:+1: nice work. 
Note that you miss the trailing space to super view constraint so that the `categoriesLabel` is truncated in this cell.
<img width="308" alt="screen shot 2016-02-15 at 9 45 20 pm" src="https://cloud.githubusercontent.com/assets/7917093/13051728/7306dbf8-d42d-11e5-9281-da960ea31101.png">

It would be better if you save the last user's choices in the filters page.

The primary challenge of this homework was to learn how to use UITableView to manage a complex form, establish communication between view controllers, and apply the correct Auto Layout constraints to build adaptive UI's that work across many different screen sizes and orientations. If you haven't already, we recommend going back and running your homework again while rotating the device / simulator to ensure your AutoLayout constraints function as expected.

We have a detailed [Project 2 Feedback Guide](http://courses.codepath.com/snippets/intro_to_ios/feedback_guides_swift/project_2_feedback.md) which covers the best practices for implementing this assignment. Read through the feedback guide point-by-point to determine ways you might be able to improve your submission. You should consider going back and implementing these improvements as well. Keep in mind that one of the most important parts of iOS development is learning the correct patterns and conventions.

If you have any particular questions about the assignment or the feedback, feel free to reply here or email us at support@codepath.com

This was a challenging assignment, congrats on completing it successfully!

**Comment by pthor** [comment]
@codepathreview  Good catch on the categories not wrapping. Fixed. I had the trailing constraint but forgot to set lines = 0 in IB.
<img width="1091" alt="screen shot 2016-02-15 at 12 34 50 pm" src="https://cloud.githubusercontent.com/assets/805796/13059672/c08bc55c-d3e0-11e5-8212-2c64da26da07.png">

  And yes, the next thing I wanted to implement was saving the last filter with User Preferences like I had in the tip app.
